osbs/core.py

```diff
@@ -405,20 +404,12 @@ def stream_logs(self, build_id):
         while True:
             buildlogs_url = self._build_url("builds/%s/log/" % build_id,
                                             **kwargs)
-            try:
-                response = self._get(buildlogs_url, stream=1,
-                                     headers={'Connection': 'close'})
-            except OsbsNetworkException as ex:
-                # pycurl reports 'empty reply from server' as
-                # FOLLOWLOCATION. Handle this as though there were no
-                # lines returned.
-                if ex.status_code != pycurl.FOLLOWLOCATION:
-                    raise
-            else:
-                check_response(response)
-                for line in response.iter_lines():
-                    last_activity = time.time()
-                    yield line
+            response = self._get(buildlogs_url, stream=1,
+                                 headers={'Connection': 'close'})
```

Comment:
The code this replaces took a while to get right. The particular case where we needed to catch the FOLLOWLOCATION error was when handling chunked data where the server only ever sends headers and finally closes the connection.

This can happen when e.g. watching logs on a OpenShift Build, when no logs are produced for 300 seconds.
https://github.com/projectatomic/osbs-client/pull/421

Does `requests` not treat this as an error?